homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -1228,7 +1228,6 @@ async def execute(self) -> None:
                 )
                 if detect_result is None:
                     # No wake word. Abort the rest of the pipeline.
-                    await self.run.end()
```

Comment:
`PipelineRun.end` is called also from the finally clause below: https://github.com/home-assistant/core/pull/100909/files#diff-a1e06b2f395e01be16e7125e181ad2aa080c8c618728e2e5aaa6bf1cd3d5a912L1293-L1296